sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2_and_later/async_samples/sample_send_request_async.py

```diff
@@ -35,53 +35,38 @@ async def sample_send_request():
         endpoint=endpoint, credential=AzureKeyCredential(key)
     )
     
-    # The `send_request` method can send custom HTTP requests that share the client's existing pipeline,
-    # while adding convenience for endpoint construction and service API versioning.
-    # Now let's use the `send_request` method to make a resource details fetching request.
-    # The URL of the request can be relative (your endpoint is the default base URL),
-    # and the API version of your client will automatically be used for the request.
-    request = HttpRequest(method="GET", url="info")
     async with client:
+        # The `send_request` method can send custom HTTP requests that share the client's existing pipeline,
+        # while adding convenience for endpoint construction and service API versioning.
+        # Now let's use the `send_request` method to make a resource details fetching request.
+        # The URL of the request can be relative (your endpoint is the default base URL),
+        # and the API version of your client will automatically be used for the request.
+        request = HttpRequest(method="GET", url="info")
         response = await client.send_request(request)
-    response.raise_for_status()
-    response_body = response.json()
-    print(
-        f"Our resource has {response_body['customDocumentModels']['count']} custom models, "
-        f"and we can have at most {response_body['customDocumentModels']['limit']} custom models."
-        f"The quota limit for custom neural document models is {response_body['customNeuralDocumentModelBuilds']['quota']} and the resource has"
-        f"used {response_body['customNeuralDocumentModelBuilds']['used']}. The resource quota will reset on {response_body['customNeuralDocumentModelBuilds']['quotaResetDateTime']}"
-    )
+        response.raise_for_status()
+        response_body = response.json()
+        print(response_body)
```

Comment:
Is there anything in the response bodies that we can reliably check for? It would be good to have an example for customers on how to use the response body. The sample originally showcased using the response, which seems more useful, would adding some checks before printing help here?